features/org.wso2.analytics.is.feature/src/main/siddhi4/Readme.rtf

```diff
@@ -0,0 +1,8 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1561\cocoasubrtf200
```

Comment:
Please make this as README.md, in markdown format.

Reply:
ok thank you